### PR TITLE
Create pull_request_template.md

### LINKED ISSUE #8: No Pull Request Template
A pull request template should be added to the Pessimism service so that developers can easily describe their work and changes in a standardized fashion.

### PR DESCRIPTION
**Pull request by the-amazing-atharva**
fixes #8

### CODE REVIEW

**Comment by epociask** [comment]
Hi @the-amazing-atharva, thank you so much for taking the time to open this PR. In the meantime, @nadir-akhtar-coinbase will be owning the review and should have some feedback shortly.

File: pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+## Fixes Issue
+
+<!-- Example: Closes #61 -->
+
+## Changes proposed
+
+<!-- List all the proposed changes in your PR -->
+
+
+## Screenshots
```
Comment (nadir-akhtar-coinbase) [inline pull_request_template.md:12]: Don't think we need a dedicated Screenshots section, as screenshots can be included as motivated to explain any proposed changes

File: pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+<!-- If your PR fixes an open issue, use `Closes #999` to link your PR with the issue. #999 stands for the issue number you are fixing -->
+
+## Fixes Issue
+
+<!-- Example: Closes #61 -->
```
Comment (nadir-akhtar-coinbase) [inline pull_request_template.md:5]:
```suggestion
<!-- Example: Closes #NNN -->
```

**Review by nadir-akhtar-coinbase — COMMENTED** [review]
Hey @the-amazing-atharva appreciate the PR, left some comments. One thought: a very helpful section would be "How did you test your change?" or equivalent. 

Moreover, potentially useful to have a way to self-categorize what kind of change is being made, but not necessary this early in the project's lifetime.

File: pull_request_template.md (hunk 1 above)
Comment (the-amazing-atharva) [reply pull_request_template.md:12]: How about I add the tag "Optional"?

Comment (nadir-akhtar-coinbase) [reply pull_request_template.md:12]: Sounds reasonable, though could be cleaner to specify this in a comment within the `Changes proposed` section. Either works

Comment (the-amazing-atharva) [reply pull_request_template.md]: @nadir-akhtar-coinbase made some changes

Thoughts??

**Review by nadir-akhtar-coinbase — APPROVED** [review]
Looks good, thanks @the-amazing-atharva!

**Review by epociask — APPROVED** [review]
LGTM

**Comment by cb-heimdall** [comment]
Review Error for epociask @ 2023-04-05 19:35:14 UTC
User failed mfa authentication, see [go/mfa-help](http://go/mfa-help)

**Review by epociask — APPROVED** [review]

**Comment by cb-heimdall** [comment]
Review Error for epociask @ 2023-04-05 19:35:31 UTC
User failed mfa authentication, see [go/mfa-help](http://go/mfa-help)

**Review by epociask — APPROVED** [review]

**Pull request merged**